src/tasks/mixin/liaison_mixin.py

```
@@ -77,6 +77,14 @@ def transfer_to_home_point(self, box=None, should_check_out_boat=False):
             box = self.box.left
         self.ensure_main()
 
+        if should_check_out_boat and getattr(self, "_daily_boat_state_confirmed", False):
+            self.log_info("已共享帝江号状态，跳过重复传送确认")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)